doc/short_vignettes/trackmutationfates.md

```diff
@@ -0,0 +1,150 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(tracking_mutation_fates)=
+
+# Need title
+
+```{code-cell} python
+---
+tags: ['hide-input']
+---
+
+import fwdpy11
+import numpy as np
+import msprime
+```
+
+```{code-cell} python
+import fwdpy11.conditional_models
+```
+
+
+```{code-cell} python
+---
+tags: ['hide-input']
+---
+
+def setup(prune_selected=False):
+    # Dropping mutations requires existing
+    # ancestry, which we can get either
+    # from a burn-in or from msprime.
+    initial_ts = msprime.sim_ancestry(
+        samples=500,
```

Comment:
Same here?

Reply:
again, will double-check.  Some of this is confusion due to the change in defaults to diploidy for sim_ancestry.